fhttp/httprunner.go

```diff
@@ -69,8 +68,8 @@ func (httpstate *HTTPRunnerResults) Run(ctx context.Context, t periodic.ThreadID
 		httpstate.aborter.Abort(false)
 		log.Infof("Aborted run because of code %d - data %s", code, DebugSummary(body, 1024))
 	}
-	if code == http.StatusOK {
-		return true, "200"
+	if code >= 200 && code <= 299 {
+		return true, fmt.Sprint(code)
```

Comment:
should be strconv.Itoa(code) (and line 74 too so a preexisting problem), also noticing line 61 (calling Fetch instead of StreamFetch is also wrong - puzzled how I missed that in #721 refactor)